src/Modules/UsersModule/components/UsersList/Users.module.css

```diff
@@ -0,0 +1,109 @@
+.responsiveTableProjects {
+  padding: 0;
+  li {
+    border-radius: 10px;
+    padding: 20px 30px;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 20px;
+  }
+  .tableHeader {
+    padding: 25px 35px;
+    /* background-color: #315951e5; */
+    background-color: var(--tableHdBg);
+    font-family: Poppins;
+    font-size: 16px;
```

Comment:
I highly encourage reading this article about using units and why using pixels for font-sizes is not good.
https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/